.circleci/config.yml

```diff
@@ -0,0 +1,204 @@
+aliases:
+  - &upgrade-git
+    name: Update Git
+    command:  |
+      git --version
+      sudo apt-get install \
+        build-essential flex bison \
+        libreadline6-dev zlib1g-dev \
+        libssl-dev \
+        libcurl4-gnutls-dev \
+        libexpat1-dev \
+        tcl tk \
+        tcl-dev gettext \
+        asciidoc \
+        docbook2x
+
+      wget https://www.kernel.org/pub/software/scm/git/git-2.13.0.tar.gz
+      tar -xvzf git-2.13.0.tar.gz
+      cd git-2.13.0
+      sudo ./configure
+      sudo make && sudo make install
+      git --version
+
+  - &setup
+    name: Setup
+    command: ./install.sh
+    when: always
+
+  - &install-ant
+    name: Install Ant
+    command: brew install ant
+
+linux: &linux
+  working_directory: ~/SalesforceMobileSDK-UITest
+  docker:
+    - image: circleci/android:api-27-node8-alpha
+  environment:
+    - TERM: "dumb"
+    - ADB_INSTALL_TIMEOUT: 15
+    - _JAVA_OPTIONS: "-XX:+UnlockExperimentalVMOptions -XX:+UseCGroupMemoryLimitForHeap"
+    - GRADLE_OPTS: '-Dorg.gradle.daemon=false -Dorg.gradle.jvmargs="-XX:+HeapDumpOnOutOfMemoryError"'
+    - ANDROID_NDK: '/opt/ndk/android-ndk-r10e'
+    - BUILD_THREADS: 2
+
+#mac: &mac
```

Comment:
All references to mac/iOS commented out because the Android will fail if the account doesn't have the mac/iOS license.